server/bin/pbench-backup-tarballs.py

```diff
@@ -102,12 +95,13 @@ def sanity_check(lb_obj, s3_obj, config, logger):
 
     if not os.path.isdir(qdir):
         logger.error(
-            'The QUARANTINE directory {}, does not resolve {} to a directory'.format(qdir, os.path.realpath(qdir)))
+            'The QUARANTINE path {}, resolves to {}'
+            'which is not a directory'.format(qdir, os.path.realpath(qdir)))
         lb_obj = None
 
     # make sure the S3 bucket exists
     try:
-        s3_obj.connector.head_bucket(Bucket='{}'.format(s3_obj.bucket_name))
+        s3_obj.head_bucket(s3_obj.bucket_name)
     except Exception:
         logger.exception(
```

Comment:
Let's get rid of the exception traceback both here and on line 79: ;log them as errors instead (and fix the gold files) - the traceback is not telling us anything useful.